global.json

```diff
@@ -1,5 +1,6 @@
 {
-    "sdk": {
-        "version": "5.0.103"
-    }
+  "sdk": {
+    "version": "5.0.103",
+    "rollForward": "latestMajor"
```

Comment:
I know in the past, we've run into breaking behaviors in patch versions. That's generally why we require specific versions to force the check with what we expect to be running on.